README.md

```diff
@@ -36,23 +36,42 @@ Read about supported commands in the [CLI documentation website](https://develop
 Usage:
   flow [command]
 
-Available Commands:
-  accounts     Utilities to manage accounts
-  blocks       Utilities to read blocks
+👋 Welcome Flow developer!
+   If you are starting a new flow project use our super commands, start by running 'flow setup'. 
+
+🔥 Super Commands
+  dev          Build your Flow project
+  flix         Execute FLIX template with a given id, name, or local filename
+  generate     Generate new boilerplate files
+  setup        Start a new Flow project
+
+📦 Flow Entities
+  accounts     Create and retrieve accounts and deploy contracts
+  blocks       Retrieve blocks
+  collections  Retrieve collections
+  events       Retrieve events
+
+💬 Flow Interactions
+  scripts      Execute Cadence scripts
+  transactions Build, sign, send and retrieve transactions
+
+🔨 Flow Tools
   cadence      Execute Cadence code
-  collections  Utilities to read collections
```

Comment:
why removing these and project commands (keys, scripts, tx, etc)?

Reply:
I think they just moved. This is a copy paste of what comes from the CLI typing `flow` so it's whatever commands exist.